go/vt/vttablet/tabletserver/schema/engine.go

```diff
@@ -52,6 +50,8 @@ import (
 	"vitess.io/vitess/go/vt/vttablet/tabletserver/connpool"
 	"vitess.io/vitess/go/vt/vttablet/tabletserver/tabletenv"
 
+	"golang.org/x/exp/maps"
+
```

Comment:
This import should go above the `vitess.io` ones.

Reply:
Got it!